pkg/download/handler.go

```diff
@@ -60,6 +61,6 @@ func getRedirectURL(base, path string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	url.Path = path
+	url.Path = filepath.Join(url.Path, path)
```

Comment:
I believe `filepath` will end up joining with backslashes instead of forward slashes, so you might want to use `path.Join` instead of `filepath.Join`

Reply:
Right you are - good catch.